src/applications/personalization/profile/mocks/endpoints/user/index.js

```diff
@@ -1392,15 +1392,15 @@ const mockErrorResponses = {
     meta: {
       errors: [
         {
-          externalService: 'Vet360',
+          externalService: 'VAProfile',
```

Comment:
As a general question:

We also have the `attributes.services` array in the user data that have vet360 listed as a service. Will that also need to change to be VAProfile?

Reply:
hm, good question. That will eventually need to be changed (we have an epic for the remaining vet360 to va profile work in this repo and vets-api) but I think we don't want to change it in this PR. The string having different casing (external service `Vet360` vs service `vet360`) makes me think `attributes.service` is something slightly different. To test this out, I changed all instances of `'vet360'` in this file to `'VAProfile'` and ran the test, `src/applications/personalization/profile/tests/e2e/contact-information/focus.after.edit.cypress.spec.js`, and it failed 😅. Further evidence that I don't want to mess with the attributes yet.

Reply:
Yea that is a good call. I will look into this more on out end as well. I know that we also have EMIS and VAProfile listed a dependency in our DownTimeNotification component usages so I'm sure we will also have to address those pieces in their own time.

Reply:
@rjohnson2011 is planning on tackling that work in https://github.com/department-of-veterans-affairs/va.gov-team/issues/75205 next sprint (in two weeks).